### PR TITLE
[fix] auth_signup_verify_email user lang

### PR DESCRIPTION
**Pull request by simahawk**
Addresses #585

### CODE REVIEW

File: auth_signup_verify_email/controllers/main.py [hunk 1]
```diff
@@ -23,6 +27,17 @@ def web_auth_signup(self, *args, **kw):
         else:
             return super(SignupVerifyEmail, self).web_auth_signup(*args, **kw)
 
+    def _get_user_lang(self):
+        """Retrieve user language."""
+        langs = http.request.env['res.lang'].search_read([], ['code'])
+        supported_langs = [
+            lang['code'] for lang in langs
+        ]
+        lang = self.DEFAULT_USER_LANG
```
Comment (simahawk) [inline auth_signup_verify_email/controllers/main.py]: @pedrobaeza maybe for this we can take the website language, what do you think?

Comment (pedrobaeza) [inline auth_signup_verify_email/controllers/main.py]: Maybe website default language here?

Comment (simahawk) [reply auth_signup_verify_email/controllers/main.py]: @pedrobaeza I commented the same but didn't submit the review comment :face_with_head_bandage:

Comment (pedrobaeza) [reply auth_signup_verify_email/controllers/main.py]: hehe, then go ahead with that language default.

**Comment by simahawk** [comment]
@pedrobaeza done

**Review by pedrobaeza — APPROVED** [review]
When CIs finish correctly

**Pull request closed without merging**